lib/cli.rb

```diff
@@ -0,0 +1,164 @@
+        user_input = STDIN.gets.chomp.strip
+        
+        actual_location = Search.all.find_by(search_location: user_input)
+        
+
+          if actual_location
+            weather_description =  Weather.find_by(search_id: actual_location.id)
+            puts "The "+"Weather".colorize(:cyan)+" in #{user_input} is "+"#{weather_description.description}".colorize(:cyan)
+            puts
+            puts "Wasn't expecting that weather, now were you."
+            puts 
+            saved_location = Search.create(search_location: user_input)
+
+            saved_location_in_history = UsersSearch.create(search_id: saved_location.id, user_id: $user.id)
+            
+          else
+            is_running = true            
+            puts
+            puts `clear`
+            puts "That City doesn't have "+"weather.".colorize(:cyan)+" WIERD. Please re-select Search "+"Weather.".colorize(:cyan) 
+          end 
+            
+          
+      elsif choice == "2" 
+        puts `clear`
+        puts 
+        UsersSearch.all.each do |history|
+          puts "Name - #{history.user.name} and Location - #{history.search.search_location}" 
+        puts 
+        end 
+        puts "Someone's being "+"drafty".colorize(:cyan)+"..."
+        puts 
+
+        # writing out my idea 
+        # if same user and same search location show up more than 3 times (iterate over searches) 
+        # then spit out puts "Notice how actual user is looking at specific location"
+  
+          # if specific_searches > 5
+          #   puts "Looks like someone has been searching #{search.search_location} a lot.."      
+          # else
+          #   puts "No particular place seems to be of interest to anyone."
+          # end  
+      elsif choice == "3" 
+        puts`clear`
+        puts 
+        UsersSearch.all.each do |history|
+          puts "ID - #{history.id}, Name - #{history.user.name} and Location - #{history.search.search_location}."  
+          end 
+          puts 
+          puts "Are you sure you want to delete a specific search? You're being stormy..." 
+          puts 
+        puts "Please type in the specific ID you wish to delete"
+        puts 
+        delete_select = STDIN.gets.chomp.strip
+          if UsersSearch.find_by(id: delete_select)
+            UsersSearch.delete(delete_select)
+            puts 
+            puts "You're shady. Like the "+"weather. ".colorize(:cyan)+"Get it?"
+            puts 
+            puts "You have deleted ID #{delete_select}. I hope you're happy."
+            puts 
+            is_running = true
+          else 
+            puts 
+            puts "Please select 3 again. The ID you typed in doesn't exist."
+            puts
+            puts "Try using your eyes and reading the screen for once..."
+            puts
+          end 
+
+      elsif choice == "4" 
+        puts `clear`
+        Search.all.each do |search|
+          puts "Location: #{search.search_location}"  
+          end 
```

Comment:
```suggestion
        end 
```
You should really get a linter to catch your formatting mistakes! If it's annoying for you to see blue lines marking up your code, it's worse for someone else to see errors that you didn't catch :)
[Rubocop](https://github.com/rubocop-hq/rubocop)